policyengine_canada/parameters/gov/provinces/bc/tax/income/credits/disability/cap.yaml

```diff
@@ -0,0 +1,15 @@
+description: British Columbia caps the disability credit at this amount.
+values:
+  2022-01-01: 13_423
+  2023-01-01: 14_228
```

Comment:
isn't this just the sum of the base and additional amount? if so i don't think we need it